### PR TITLE
feat: implement message signing

### PR DESCRIPTION
**Pull request by brenopolanski**
## Summary

Task: https://trello.com/c/e8pcMUhN/108-implement-message-signing

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

**Comment by faustbrian** [comment]
@brenopolanski this needs E2E tests

**Comment by faustbrian** [comment]
@brenopolanski failing e2e tests after merging, related to this PR

**Comment by faustbrian** [comment]
@brenopolanski conflicts

**Comment by faustbrian** [comment]
@brenopolanski conflicts and still failing e2e

File: src/domains/wallet/pages/WalletDetails/__snapshots__/WalletDetails.test.tsx.snap [hunk 1]
```diff
@@ -1273,7 +1273,7 @@ exports[`WalletDetails should delete wallet 1`] = `
                       role="columnheader"
                     >
                       <div
-                        class="flex flex-inline align-top undefined"
+                        class="flex align-top flex-inline undefined"
```
Comment (goga-m) [inline src/domains/wallet/pages/WalletDetails/__snapshots__/WalletDetails.test.tsx.snap:1276]: @brenopolanski in order this undefined to be avoided, I think if you provide an empty string when `column.ClassName` is empty here: https://github.com/ArkEcosystem/desktop-wallet/blob/4b6426b4bba1d133ef9858e652334f047725a8ec/src/app/components/Table/Table.tsx#L63
Would fix this issue

Comment (goga-m) [reply src/domains/wallet/pages/WalletDetails/__snapshots__/WalletDetails.test.tsx.snap]: Update: It is now patched in a separate PR: https://github.com/ArkEcosystem/desktop-wallet/pull/2540/

Comment (brenopolanski) [reply src/domains/wallet/pages/WalletDetails/__snapshots__/WalletDetails.test.tsx.snap]: Fixed here: https://github.com/ArkEcosystem/desktop-wallet/pull/2540

**Pull request merged**